pkg/ts/query.go

```diff
@@ -1136,7 +1140,7 @@ func getExtractionFunction(agg tspb.TimeSeriesQueryAggregator) (extractFn, error
 	case tspb.TimeSeriesQueryAggregator_MIN:
 		return (roachpb.InternalTimeSeriesSample).Minimum, nil
 	}
-	return nil, errors.Errorf("query specified unknown time series aggregator %s", agg.String())
+	return nil, errors.Errorf("query specified unknown time series downsampler %s", agg.String())
```

Comment:
It's an existing problem, but the function comments on `getExtractionFunction` and `getDownsampleFunction` are incomplete.

Reply:
Well, they do return, so at least they're not incorrect! Removed the stub comments, they are unnecessary.